src/Support/SupportData.php

```diff
@@ -247,8 +247,7 @@ public function get_site_info() {
 		$loopback_status = '';
 
 		if ( class_exists( 'WP_Site_Health' ) ) {
-			$site_health     = method_exists( 'WP_Site_Health', 'get_instance' ) ? WP_Site_Health::get_instance() : new WP_Site_Health();
-			$loopback_status = $site_health->can_perform_loopback();
+			$loopback_status = ( new WP_Site_Health() )->can_perform_loopback();
```

Comment:
Even though this is wrongly flagged by PHPStan as an error but I think it's in better shape now. On the support page, there will be no instance of `WP_Site_Health` and it will be freshly instantiated anyway.

So IMO it's better to instantiate and read the required data since we will not be using this class instance anywhere else.

Reply:
Yes, this makes sense. And `WP_Site_Health::get_instance()` is the singleton static getter, so it'll instantiate it if it doesn't already exist, and only one copy will ever only ever get constructed.

Reply:
Nevermind, my comment is saying the opposite.

The `WP_Site_Health::get_instance()` method was introduced in WordPress 5.4 via https://github.com/WordPress/wordpress-develop/commit/637b6f545676ef263957263345b032ed14d45744. So that's why we have the `method_exists` check. 

The only potential problem here is that the [constructor](https://developer.wordpress.org/reference/classes/wp_site_health/__construct/) for the class has side effects. But as you say, this class wouldn't have been instantiated on the support page anyway.